src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -460,15 +528,23 @@ export class MgtPersonCard extends MgtTemplatedComponent {
         this.personDetails = (parent as MgtPerson).personDetails;
         this.personImage = (parent as MgtPerson).personImage;
       }
-    }
+    } else if (this.userId || this.personQuery === 'me') {
+      // Use userId or 'me' query to get the person and image
+      const person = await getUserWithPhoto(graph, this.userId);
 
-    if (this.personDetails) {
-      return;
-    }
+      this.personDetails = person;
+      this.personImage = this.getImage();
+    } else if (this.personQuery) {
+      // Use the personQuery to find our person.
+      const people = await findPerson(graph, this.personQuery);
 
-    const provider = Providers.globalProvider;
-    if (!provider || provider.state !== ProviderState.SignedIn) {
-      return;
+      if (people && people.length) {
+        this.personDetails = people[0];
+
+        if (this.personImage === '@') {
+          this.loadImage();
+        }
```

Comment:
I think this is causing the issue with the image not showing when person-query is used. Should probably be:

```suggestion
          this.loadImage();
```

Could you make the same fix for person?

Reply:
On it

Reply:
Done